docs(README): Add method to calendar in quickstart example (#535)
sebbo2002/ical-generator: README.md

import http from 'node:http';

const calendar = ical({name: 'my first iCal'});

> // A method is required for outlook to display the invitation properly

Reviewer: ```suggestion
// A method is required for outlook to display event as an invitation
```

Since you don't always want to display an invitation, I would rather word it that way. In most cases that I have seen, only entries in the calendar should be displayed. Without rendering an invitation for it.

Author: Yes, sure! You would have a better idea of how others use your package. I just want to save people some bug hunting when trying to display the event as an invitation.
